Techtonica/keyboard-shortcuts-practice
# Fixes #73 Integrate Auth0 middleware into the server

- Updated the README with instructions to set up the Auth0 along with environment variables in .env.example
- Configured the Auth0 middleware when the Auth0 variables are given
Link to [Issue](https://github.com/Techtonica/keyboard-shortcuts-practice/issues/73)


To Test:
- Follow the instructions in the README to setup Auth0 and integrate
- To test the logging go to `http://localhost:3000/login`. It should render a `Sign In with Auth0` screen, and you must create an account with your Web Application
(Note: if you get a BadRequestError: check.state missing, it is an error with the Cookies. Go back to `http://localhost:3000` and log in and you should notice you are already logged in)
- To test the logging out go to `http://localhost:3000/logout`, when inspecting the `Network` you should see the logout network call and the screen will refresh
- After logging out, you can log back in and see the `Sign In with Auth0 `again
